internal/controller/policyexception_controller_test.go

```diff
@@ -0,0 +1,174 @@
+/*
+Copyright 2023.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package controller
+
+import (
+	"context"
+
+	policyAPI "github.com/giantswarm/policy-api/api/v1alpha1"
+	kyvernov1 "github.com/kyverno/kyverno/api/kyverno/v1"
+	kyvernov2beta1 "github.com/kyverno/kyverno/api/kyverno/v2beta1"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	apiextv1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/client-go/kubernetes/scheme"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/log"
+	"sigs.k8s.io/controller-runtime/pkg/log/zap"
+)
+
+var _ = Describe("Converting GSPolicyException to Kyverno Policy Exception", func() {
+	var (
+		ctx                    context.Context
+		kyvernoClusterPolicy   kyvernov1.ClusterPolicy
+		gsPolicyException      policyAPI.PolicyException
+		r                      *PolicyExceptionReconciler
+		kyvernoPolicyException kyvernov2beta1.PolicyException
+	)
+
+	BeforeEach(func() {
+
+		// We initialize the Policy Exception Reconciler first.
```

Comment:
Nitpick about these comments: it's obvious what's going on on the line below, so there's no real need to explain it. A general rule of thumb for comments is to not explain _what_ you're doing, but _why_ you're doing it. The code should tell you _what_ it's doing upfront.

Reply:
Thanks for the tip, I'll keep this in mind for in the future!